app/helpers/compute_resources_vms_helper.rb

```diff
@@ -263,23 +263,24 @@ def vm_import_action(vm, html_options = {})
         :type => 'unmanaged'),
       html_options
     )
-
     import_managed_link + import_unmanaged_link
   end
 
-  def vm_associate_action(vm)
-    display_link_if_authorized(
-      _("Associate VM"),
-      hash_for_associate_compute_resource_vm_path(
-        :compute_resource_id => @compute_resource,
-        :id => vm.identity
-      ).merge(
-        :auth_object => @compute_resource,
-        :permission => 'edit_compute_resources'),
-      :title => _("Associate VM to a Foreman host"),
-      :method => :put,
-      :class => "btn btn-default"
-    )
+  def vm_associate_action(vm, isdropdown = false)
```

Comment:
Nit: The interface of this method is hard to use. `vm_associate_action(VM, false)` -> It's hard to figure out what `false` means.

Reply:
@timogoebel Thanks! do you have a better proposition? (:

Reply:
@adiabramovitch: Yep, I'd like to recommend this: https://thoughtbot.com/blog/ruby-2-keyword-arguments